Added support for (deprecated) VWR sentence

joabakk opened the pull request:
Relative wind angle and speed

tkurki commented on codecs/VWR.js:
```diff
@@ -0,0 +1,74 @@
+/*
+* RPM codec
```
These comments don't really serve a purpose I think and can be omitted, as they are frequently forgotten.

joabakk commented:
Updated the comment. However I agree they could be removed from all sentences, and replaced by a reference to the project and the location, if anything
